accounts/abi/abi.go

```diff
@@ -193,8 +173,8 @@ var (
 	r_byte       = reflect.TypeOf(byte(0))
 )
 
-// unmarshal output in v according to the abi specification
-func (abi ABI) unmarshal(v interface{}, name string, output []byte) error {
+// Unpack output in v according to the abi specification
+func (abi ABI) Unpack(v interface{}, name string, output []byte) error {
```

Comment:
The reason I didn't do in a separate PR was that it's simple enough and it would have just needed an extra round of reviews and rebases to get merged in. If you want though I can easily separate out these changes and open an explicit PR with them.